sms/src/main/java/org/motechproject/sms/web/IncomingController.java

```diff
@@ -88,28 +83,52 @@ public void handleIncoming(@PathVariable String configName, @RequestParam Map<St
         }
         Template template = templateService.getTemplate(config.getTemplateName());
 
+        eventRelay.sendEventMessage(inboundEvent(config.getName(),
+                getSenderKey(params, template),
```

Comment:
the naming is wrong, these do not return keys but actual values, they Key part should get removed